libhb/encavcodec.c

```diff
@@ -228,6 +228,16 @@ int encavcodecInit( hb_work_object_t * w, hb_job_t * job )
                     break;
             }
         }break;
+        case AV_CODEC_ID_AV1:
+        {
+            switch (job->vcodec) {
+                case HB_VCODEC_FFMPEG_NVENC_AV1:
+                case HB_VCODEC_FFMPEG_NVENC_AV1_10BIT:
+                    hb_log("encavcodecInit: AV1 (Nvidia NVENC)");
+                    codec_name = "av1_nvenc";
+                    break;
```

Comment:
leaving the switch statement, since I expect more FFmpeg AV codec in near future.